tests/data/Block.0.sample1.json

```diff
@@ -1,126 +1,130 @@
-        "missing_validators": []
-    },
-    "txs": [
-        {
-            "type": 1,
-            "inputs": [],
-            "outputs": [
-                {
-                    "value": "20000000000000",
-                    "address": "GDNODE2IMTDH7SZHXWDS24EZCMYCEJMRZWB3S4HLRIUP6UNGKVVFLVHQ"
-                },
-                {
-                    "value": "20000000000000",
-                    "address": "GDNODE3EWQKF33TPK35DAQ3KXAYSOT4E4ACDOVJMDZQDVKP66IMJEACM"
-                },
-                {
-                    "value": "20000000000000",
-                    "address": "GDNODE4KTE7VQUHVBLXIGD7VEFY57X4XV547P72D37SDG7UEO7MWOSNY"
-                },
-                {
-                    "value": "20000000000000",
-                    "address": "GDNODE5T7TWJ2S4UQSTM7KDHU2HQHCJUXFYLPZDDYGXIBUAH3U3PJQC2"
-                },
-                {
-                    "value": "20000000000000",
-                    "address": "GDNODE6ZXW2NNOOQIGN24MBEZRO5226LSMHGQA3MUAMYQSTJVR7XT6GH"
-                },
-                {
-                    "value": "20000000000000",
-                    "address": "GDNODE7J5EUK7T6HLEO2FDUBWZEXVXHJO7C4AF5VZAKZENGQ4WR3IX2U"
-                }
-            ],
-            "payload": ""
-        },
-        {
-            "type": 0,
-            "inputs": [],
-            "outputs": [
-                {
-                    "value": "610000000000000",
-                    "address": "GCOQEOHAUFYUAC6G22FJ3GZRNLGVCCLESEJ2AXBIJ5BJNUVTAERPLRIJ"
-                },
-                {
-                    "value": "610000000000000",
-                    "address": "GCOQEOHAUFYUAC6G22FJ3GZRNLGVCCLESEJ2AXBIJ5BJNUVTAERPLRIJ"
-                },
-                {
-                    "value": "610000000000000",
-                    "address": "GCOQEOHAUFYUAC6G22FJ3GZRNLGVCCLESEJ2AXBIJ5BJNUVTAERPLRIJ"
-                },
-                {
-                    "value": "610000000000000",
-                    "address": "GCOQEOHAUFYUAC6G22FJ3GZRNLGVCCLESEJ2AXBIJ5BJNUVTAERPLRIJ"
-                },
-                {
-                    "value": "610000000000000",
-                    "address": "GCOQEOHAUFYUAC6G22FJ3GZRNLGVCCLESEJ2AXBIJ5BJNUVTAERPLRIJ"
-                },
-                {
-                    "value": "610000000000000",
-                    "address": "GCOQEOHAUFYUAC6G22FJ3GZRNLGVCCLESEJ2AXBIJ5BJNUVTAERPLRIJ"
-                },
-                {
-                    "value": "610000000000000",
-                    "address": "GCOQEOHAUFYUAC6G22FJ3GZRNLGVCCLESEJ2AXBIJ5BJNUVTAERPLRIJ"
-                },
-                {
-                    "value": "610000000000000",
-                    "address": "GCOQEOHAUFYUAC6G22FJ3GZRNLGVCCLESEJ2AXBIJ5BJNUVTAERPLRIJ"
-                }
-            ],
-            "payload": ""
-        }
+    "header":{
```

Comment:
Please print out JSON Pretty and make it easier to compare with previous data.
https://jsonformatter.org/json-pretty-print